src/listeners/message.ts

```diff
@@ -83,32 +83,91 @@ client.on("message", async message => {
     const member = await message.client.guilds.cache.get("549503328472530974")!.members.fetch(message.author.id)
     if (!message.content.startsWith(prefix) && message.author !== client.user && message.channel.type === "dm" && !member!.roles.cache.has("645208834633367562")) { // Muted
         const staffBots = client.channels.cache.get("624881429834366986") as Discord.TextChannel
-        const staffMsg = new Discord.MessageEmbed()
-            .setColor(neutralColor)
-            .setAuthor("Incoming message from " + message.author.tag)
-            .setDescription(message.content)
-            .addField("To reply", `\`+dm ${message.author.id} \``)
-
-        const dmEmbed = new Discord.MessageEmbed()
-            .setColor(successColor)
-            .setAuthor(getString("outgoing", "global"))
-            .setDescription(message.content)
-            .setFooter(getString("outgoingDisclaimer", "global"))
-        if (message.attachments.size > 1) {
-            const images: (Discord.BufferResolvable | Stream)[] = []
-            message.attachments.forEach(file => images.push(file.attachment))
-            staffMsg.setTitle("View attachments")
-            dmEmbed.setTitle("Attachments sent")
-            staffBots.send({ embed: staffMsg, files: images })
+        const timeToPass = 172800,
+        confirmTime = 60 // in seconds
+        if (!author.staffMsgTimestamp || author.staffMsgTimestamp - timeToPass > message.createdTimestamp) {
```

Comment:
```suggestion
        if (!author.staffMsgTimestamp || author.staffMsgTimestamp + timeToPass * 1000 < message.createdTimestamp) {
```
If I remember correctly, `message.createdTimestamp` gives timestamp in milliseconds, but `timeToPass` in seconds.
An incorrect comparison was also made.